documentation/modules/auxiliary/gather/snare_registry.md

```diff
@@ -0,0 +1,105 @@
+## Description
+
+  This module uses the Registry Dump feature of the [Snare Lite for Windows](https://sourceforge.net/projects/snare/) HTTP service on 6161/TCP to retrieve the Windows registry. The Dump Registry functionality is unavailable in Snare Enterprise.
```

Comment:
I'll fix this. The documentation and web interface refers to it as "Registry Dump". I'm not sure why I called it "Dump Registry."